components/ome-xml/src/ome/units/UNITS.java

```diff
@@ -66,7 +66,7 @@ public String getName()
   public static final ome.units.unit.Unit<ElectricPotential> VOLT    = ome.units.unit.Unit.<ElectricPotential>CreateBaseUnit("SI.VOLT", "V");
   public static final ome.units.unit.Unit<Frequency>         HERTZ   = ome.units.unit.Unit.<Frequency>CreateBaseUnit("SI.HERTZ", "Hz");
   public static final ome.units.unit.Unit<Length>            METRE   = ome.units.unit.Unit.<Length>CreateBaseUnit("SI.METRE", "m");
-  public static final ome.units.unit.Unit<Length>            INCH    = ome.units.unit.Unit.<Length>CreateBaseUnit("Imperial.INCH", "in");
+  public static final ome.units.unit.Unit<Length>            INCH    = METRE.multiply(0.0254).setSymbol("in");
```

Comment:
Maybe I'm missing something, but wouldn't adding a ```setMeasurementSystem(String)``` method to ```ome.units.unit.Unit``` (similar to ```setSymbol(String)```) be enough?  Then you could do ```METRE.multiply(0.0254).setMeasurementSystem("Imperial").setSymbol("in")```, and I wouldn't think any other changes would be necessary.